tests/unit/AggregateVersionTest.php

```diff
@@ -36,7 +36,7 @@ public static function provideNegativeIntegerSet(): iterable
     public function throws_if_constructed_with_a_less_than_or_equal_to_1(int $lessThanOne): void
     {
         $this->expectException(\InvalidArgumentException::class);
-        $this->expectExceptionMessage('Aggregate version must be >= 1.');
+        $this->expectExceptionMessage('Aggregate version must be >= 1,');
```

Comment:
```suggestion
        $this->expectExceptionMessage(\sprintf('Aggregate version must be >= 1, %d given.', $lessThanOne));
```

maybe still assert overall message?